### ISSUE TITLE
[FEATURE] Override Spell Preparation Calculation

### ISSUE THREAD

**Issue by CepheiA2** [voodoofrog/vf-5e-sheet-addons#19]
Hi. I'm a user of your module and have been thoroughly enjoying its features for managing spells.
I would like to request the addition of a feature that allows users to override the default spell preparation calculation, with a manual flat number for example.

The issue arises particularly when using many custom or modified classes, as the current feature tends to encounter bugs or fails to function altogether in these cases.

Here are a few potential use cases for such a feature:

- Players who are using homebrew or modified spellcasting rules may need to adjust the spell preparation calculation to align with these rules.
- Game masters running custom campaigns may want the ability to tailor the spell preparation process to fit the unique magic system of their setting.
- Some players may simply prefer a different calculation method for personal preference or playstyle reasons.

**Comment by voodoofrog** [comment]
I'll consider it, but as it stands the prep limits are coming from the classes themselves though, this module's prep feature was originally intended to simply reflect what is there in the data.

**Comment by Jadowo** [comment]
> I'll consider it, but as it stands the prep limits are coming from the classes themselves though, this module's prep feature was originally intended to simply reflect what is there in the data.

Yes pls, I use custom spell preperation with spell points. It would be nice <3
